config/jobs/kubernetes/sig-network/ci-e2e-gce-netd.yaml

```diff
@@ -597,10 +596,9 @@ periodics:
     containers:
     - args:
       - "--timeout=340"
-      - "--repo=k8s.io/kubernetes=release-1.11"
```

Comment:
you need --bare, like [these jobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/gke/gke-staging.yaml#L9-L13) (presubmit should catch this ?!)